website/content/en/blog/clusterlink-intro/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Introducing ClusterLink: Simplifying Multicluster Service Connectivity"
+linkTitle: Introducing ClusterLink
+date: 2024-04-29
+author: Etai Lev Ran
+type: blog
+draft: true
+---
+
+{{% imgproc "field" Fill "800x250" /%}}
+
+Deploying microservice applications across multiple clusters offers many benefits.
+ These include, for example, improved fault tolerance, increased scalability, performance
+ or regulatory compliance. In addition, use of multiple locations may be required to
+ access specialized hardware, services or data sources available only in those
+ locations. Kubernetes SIG multicluster [lists](https://multicluster.sigs.k8s.io/#problem-statement-why-multicluster)
+ additional reasons for using multiple clusters.
+
+When attempting to realize the benefits of multicluster applications, at least two
+ aspects must be addressed: orchestration (sometimes referred to as *"scheduling"*)
+ and connectivity. There are several existing open source projects providing these
+ capabilities. For example, projects such as [Open Cluster Management](https://open-cluster-management.io/)
+ and [KubeStellar](https://kubestellar.io) simplify orchestration and management of
+ workloads across multiple clusters.
+
+In order to facilitate cross cluster communications, you could use Kubernetes
+ native resources, including [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+ and [Gateway API](https://kubernetes.io/docs/concepts/services-networking/gateway/).
+ Or, alternatively, chose from a number of existing open source projects, such
+ as [Istio](https://istio.io), [Skupper](https://skupper.io) and [Submariner](https://submariner.io/).
+ By and large, these solutions attempt to conjoin the multiple clusters, flattening the
+ isolated networks into a single flat "mesh" shared between the connected clusters.
+ The goal is, to the extent possible, extend the Kubernetes single cluster network
+ abstraction to multicluster use cases.
+
+The creation of a shared mesh is not always desirable and may place additional
+ constraints on administrators, developers and the workloads they manage. For example,
+ it might make assumptions on objects, such as Services, being the "same" based
+ on the objects sharing a name (i.e., namespace sameness as defined
+ [here](https://istio.io/latest/docs/ops/configuration/traffic-management/multicluster/)
+ or [here](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)),
+ or require planning in assignment of IP addresses across independent clusters, etc.
```

Comment:
```suggestion
 or require planning in assignment of IP addresses across independent clusters.
```
[No need for "etc." when using "For example"](https://english.stackexchange.com/questions/52750/is-it-correct-that-etc-can-not-be-used-together-in-a-sentence-with-for-examp)